dbt_copilot_helper/commands/dns.py

```diff
@@ -741,5 +746,212 @@ def assign(app, domain_profile, project_profile, svc, env):
         return
 
 
+@domain.command()
+@click.option(
+    "--project-profile", help="AWS account profile name for certificates account", required=True
+)
+@click.option("--env", help="AWS Copilot environment name", required=True)
+@click.option("--app", help="Application Name", required=True)
+@click.option("--svc", help="Service Name", required=True)
+@click.option("--delete", help="Delete the CDN domain", is_flag=True)
+# Feature to be added at later date if needed, there shouldn't be a need to remove all domains
+# @click.option("--force", help="Force remove", is_flag=True)
+def cdn(project_profile, env, app, svc, delete, force=False):
+    """Assigns a CDN domain name to application loadbalancer."""
+    project_session = get_aws_session_or_abort(project_profile)
+    elb_client = project_session.client("elbv2")
+    acm_client = project_session.client("acm")
+
+    response = elb_client.describe_listeners(
+        LoadBalancerArn=get_load_balancer_configuration(project_session, app, svc, env)[
+            "LoadBalancers"
+        ][0]["LoadBalancerArn"],
+    )
+
+    # Certificates and domains only need to be configured on the https listener.
+    for listener in response["Listeners"]:
+        if listener["Protocol"] == "HTTPS":
+            response = elb_client.describe_rules(
+                ListenerArn=listener["ListenerArn"],
+            )
+            # If there are multiple services there will be more than one rule.
+            if len(response["Rules"]) > 2:
```

Comment:
Should this be...
```suggestion
            if len(response["Rules"]) > 1:
```

Reply:
updated comment in code.  Reason for >2, one is default and one is our custom domain.  So if  > 2 then we know multiple domains have been configured.